src/frontend/components/TranscriptSentence/TranscriptSentence.tsx

```diff
@@ -0,0 +1,28 @@
+import { Text } from 'grommet';
+import * as React from 'react';
+import styled from 'styled-components';
+import { VTTCue } from 'vtt.js';
+
+export const ActiveSentence = styled(Text)`
+  background-color: rgba(242, 94, 35, 0.25);
+  outline: 1px solid rgba(242, 94, 35, 0.5);
+`;
+
+interface TranscriptSentenceProps {
+  cue: VTTCue;
+  active: boolean;
+}
+
+export class TranscriptSentence extends React.Component<
```

Comment:
Should just be an arrow function since you don't use any lifecycle method

Reply:
Done